File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/FeeSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class FeeSearch extends AbstractEntity {
	String admissionNumber;
	String studentName;
	@Temporal(TemporalType.DATE)
	Date feeCollectionDate;
}

File: scholar-web/src/main/webapp/app/model/finance/fees/FeeSearch.js
Ext.define('scholar.model.finance.fees.FeeSearch', {
	extend: 'Ext.data.Model',
	fields : [{
		name : 'id',
		type : 'int'
	},  {
		name : 'admissionNumber',
		type : 'string'
	}, {
		name : 'studentName',
		type : 'string'
	}, {
		name : 'feeCollectionDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ],
	
});